number_of_neurons: [10, 10, 10]
distance_between_neurons: [ 0.25, 0.25, 0.25]


#standard deviation of recurrent connections. Expressed in number of cells
recurrent_connection_std: 3

input_learning_rate: 0.75

min_input_age: 5

#local view activation: multiple or single
local_view_activation_: 'single'


#weighs function type: mexican_hat or gaussian
weight_function: 'mexican_hat'



